translations/ja.json

```
@@ -426,11 +426,11 @@
-	"Output format. Accepted values: [json, yaml]": "",
+	"Output format. Accepted values: [json, yaml]": "出力フォーマット。許容値: [json, yaml]",
 	"Outputs minikube shell completion for the given shell (bash, zsh or fish)\n\n\tThis depends on the bash-completion binary.  Example installation instructions:\n\tOS X:\n\t\t$ brew install bash-completion\n\t\t$ source $(brew --prefix)/etc/bash_completion\n\t\t$ minikube completion bash \u003e ~/.minikube-completion  # for bash users\n\t\t$ minikube completion zsh \u003e ~/.minikube-completion  # for zsh users\n\t\t$ source ~/.minikube-completion\n\t\t$ minikube completion fish \u003e ~/.config/fish/completions/minikube.fish # for fish users\n\tUbuntu:\n\t\t$ apt-get install bash-completion\n\t\t$ source /etc/bash_completion\n\t\t$ source \u003c(minikube completion bash) # for bash users\n\t\t$ source \u003c(minikube completion zsh) # for zsh users\n\t\t$ minikube completion fish \u003e ~/.config/fish/completions/minikube.fish # for fish users\n\n\tAdditionally, you may want to output the completion to a file and source in your .bashrc\n\n\tNote for zsh users: [1] zsh completions are only supported in versions of zsh \u003e= 5.2\n\tNote for fish users: [2] please refer to this docs for more details https://fishshell.com/docs/current/#tab-completion\n": "指定されたシェル用の minikube シェル補完コマンドを出力 (bash、zsh、fish)\n\n\tbash-completion バイナリーに依存しています。インストールコマンドの例:\n\tOS X:\n\t\t$ brew install bash-completion\n\t\t$ source $(brew --prefix)/etc/bash_completion\n\t\t$ minikube completion bash \u003e ~/.minikube-completion  # bash ユーザー用\n\t\t$ minikube completion zsh \u003e ~/.minikube-completion  # zsh ユーザー用\n\t\t$ source ~/.minikube-completion\n\t\t$ minikube completion fish \u003e ~/.config/fish/completions/minikube.fish # fish ユーザー用\n\tUbuntu:\n\t\t$ apt-get install bash-completion\n\t\t$ source /etc/bash_completion\n\t\t$ source \u003c(minikube completion bash) # bash ユーザー用\n\t\t$ source \u003c(minikube completion zsh) # zsh ユーザー用\n\t\t$ minikube completion fish \u003e ~/.config/fish/completions/minikube.fish # fish ユーザー用\n\n\tさらに、補完コマンドをファイルに出力して .bashrc 内で source を実行するとよいでしょう\n\n\t注意 (zsh ユーザー): [1] zsh 補完コマンドは zsh バージョン \u003e= 5.2 でのみサポートしています\n\t注意 (fish ユーザー): [2] 詳細はこちらのドキュメントを参照してください https://fishshell.com/docs/current/#tab-completion\n",
 	"Overwrite image even if same image:tag name exists": "同じ image:tag 名が存在していてもイメージを上書きします",
 	"Path to the Dockerfile to use (optional)": "使用する Dockerfile へのパス (任意)",
-	"Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share": "",
+	"Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share": "qemu ファームウェアファイルへのパス。デフォルト: Linux の場合、デフォルトのファームウェアの場所。macOS の場合、 brew のインストール場所。Windows の場合、C:\\Program Files\\qemu\\share",
```

```suggestion
	"Path to the qemu firmware file. Defaults: For Linux, the default firmware location. For macOS, the brew installation location. For Windows, C:\\Program Files\\qemu\\share": "qemu ファームウェアファイルへのパス。デフォルト: Linux の場合、デフォルトのファームウェアの場所。macOS の場合、brew のインストール場所。Windows の場合、C:\\Program Files\\qemu\\share",
```
The space in front of "brew" that I suggested deleting is still remained.

---

done